public/js/monitor.js

```diff
@@ -434,14 +483,15 @@ function resizeDashboardMargin() {
   const dropDownMenu = document.querySelector(".mobile-nav.show-mobile");
   dropDownMenu.addEventListener("click", () => toggleDropDownMenu(dropDownMenu));
 
-
-
   const acceptedLanguages = navigator.languages;
   const acceptedFirstLanguageIsEnglish = acceptedLanguages[0].includes("en");
 
   if (!acceptedFirstLanguageIsEnglish && document.getElementById("fxaCheckbox")) {
-    document.getElementById("fxaCheckbox").style.display = "none";
-    return;
+    // Growth Experiment -This overrides the language filter if they've been sorted into the treatment cohort.
+    if (document.body.dataset.experiment !== "vb" || !document.body.dataset.experiment) {
+      document.getElementById("fxaCheckbox").style.display = "none";
+      return;
+    }
```

Comment:
This is the crux of the experiment – if the user is on branch VB, show the checkbox.